roles/confluent.test/molecule/rbac-plain-custom-rhel-secret/molecule.yml

```diff
@@ -0,0 +1,302 @@
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: kafka-broker3
+    hostname: kafka-broker3.confluent
+    groups:
+      - kafka_broker
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: schema-registry1
+    hostname: schema-registry1.confluent
+    groups:
+      - schema_registry
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: kafka-connect1
+    hostname: kafka-connect1.confluent
+    groups:
+      - kafka_connect
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: ksql1
+    hostname: ksql1.confluent
+    groups:
+      - ksql
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: control-center1
+    hostname: control-center1.confluent
+    groups:
+      - control_center
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    published_ports:
+      - "9021:9021"
+    networks:
+      - name: confluent
+  - name: kafka-rest1
+    hostname: kafka-rest1.confluent
+    groups:
+      - kafka_rest
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+provisioner:
+  name: ansible
+  config_options:
+    defaults:
+      hash_behaviour: merge
+  playbooks:
+    converge: ../../../../all.yml
+  inventory:
+    group_vars:
+      all:
+        scenario_name: rbac-plain-custom-rhel-secret
+
+        ssl_enabled: true
+        sasl_protocol: plain
+
+        ssl_custom_certs: true
+        # Paths relative to all.yml
+        ssl_ca_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/ca.crt"
+        ssl_signed_cert_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
+        ssl_key_filepath: "roles/confluent.test/molecule/{{scenario_name}}/generated_ssl_files/{{inventory_hostname}}-key.pem"
+        ssl_key_password: keypass
+
+        rbac_enabled: true
+
+        # Encrypt all LDAP passwords, in all config files
+        secrets_protection_enabled: true
```

Comment:
Hey thanks for this! Instead of a whole new scenario can we just add secrets protection to the existing rbac-plain-custom-rhel scenario? we already have soo many scenarios. 

This would entail moving these vars over to the other molecule.yml and then move the verify tasks as well?

Reply:
Domenic - There is no rbac-plain-custom-rhel.
It's something I created as it's a setup one of my prospects was having severe issues setting up. I haven't checked it in yet is all... so yes, just accept this PR and you'll have a "rbac-plain-custom-rhel" scenario ;-)

Reply:
sorry, you are right! The intention of my comment was not to add one more scenario to our list. Each additional scenario increases the amount of time it takes us to test for a release.
 
@JumaX wdyt about changing rbac-scram-provided-debian to rbac-plain-provided-secret-debian ? It looks like we have two scram rbac scenarios rn?

Reply:
Yeah it makes sense to consolidate them, they are almost dupes of each other.  Let's move this into that existing scenario.